### PR TITLE
Fix return type (int → bool)

### PR DESCRIPTION
**Pull request by msamesch**
Mentioned in doc-de/issues/67 by Daijobou (thanks).
Changelog entry?

### CODE REVIEW

**Comment by Daijobou** [comment]
Shouldn't it be mentioned as a hint that before 3.6.0 it was an INTEGER? ;)

I found in kohana framework
`if ($watermark->getImageAlphaChannel() !== Imagick::ALPHACHANNEL_ACTIVATE)`

Source https://github.com/koseven/koseven/blob/master/modules/image/classes/Kohana/Image/Imagick.php#L203

So this is now with version Imagick v3.6.0 a _bug_ in this framework (and maybe others).

> imagick::ALPHACHANNEL_ACTIVATE (int)
> This constant is available if Imagick has been compiled against ImageMagick version 6.3.8 or higher.

Source https://www.php.net/manual/en/imagick.constants.php

**Comment by msamesch** [comment]
> Shouldn't it be mentioned as a hint that before 3.6.0 it was an INTEGER? ;) So that developers understand what happened here?

Yes, but I wasn't sure about the exact changelog entry.

I tagged the PR 'help wanted' now.

**Comment by saundefined** [comment]
> Yes, but I wasn't sure about the exact changelog entry.

Not sure, but maybe just something like that?

```xml
<row>
  <entry>imagemagick 3.6.0</entry>
  <entry>
    Returns a &boolean; now; previously, an &integer; was returned.
  </entry>
</row>
```

**Comment by msamesch** [comment]
Hi Sergey,
overlapping comment/edited comment ;)
Thanks for the snippet (added) and the hint. So it's not only about the PHP version.
Which are the conditions for true/false?
Maybe it's better if you or sb else completes this commit.

**Comment by cmb69** [comment]
Maybe @Danack can clarify?

**Review by msamesch — COMMENTED** [review]
I used the source. ;)

File: reference/imagick/imagick/getimagealphachannel.xml [hunk 1]
```diff
@@ -9,14 +9,12 @@
...
-   <modifier>public</modifier> <type>int</type><methodname>Imagick::getImageAlphaChannel</methodname>
+   <modifier>public</modifier> <type>bool</type><methodname>Imagick::getImageAlphaChannel</methodname>
    <void />
   </methodsynopsis>
 
   <para>
-   Gets the image alpha channel value. The returned value is one of the 
-   <link linkend="imagick.constants.alphachannel">alpha channel constants</link>.
-   &imagick.method.available.0x640;
+   Gets the image alpha channel value.
```
Comment (cmb69) [inline reference/imagick/imagick/getimagealphachannel.xml:17]: I think this is somewhat misleading; maybe better:
```suggestion
   Returns whether the image has an alpha channel.
```

**Review by cmb69 — COMMENTED** [review]
Thank you! This looks mostly good to me, to there is one issue that could be addressed.

**Comment by msamesch** [comment]
Thanks for the review and the correction.

**Pull request merged**